js/stores/eventStore.js

```diff
@@ -54,7 +54,7 @@ const addPageView = (url, tabId) => {
     tab.isDestroyed() ||
     !tab.session.partition.startsWith('persist:')
 
-  if (url && isSourceAboutUrl(url) || isPrivate) {
+  if (url && (isSourceAboutUrl(url) || isPrivate)) {
```

Comment:
i think this should be `(url && isSourceAboutUrl(url)) || isPrivate`, assuming the previous code was correct